spec/controllers/idv/in_person/verify_info_controller_spec.rb

```diff
@@ -116,6 +117,129 @@
         )
       end
     end
+
+    context 'tracks costs' do
+      let(:review_status) { 'pass' }
+      let(:async_state) { instance_double(ProofingSessionAsyncResult) }
+      let(:adjudicated_result) do
+        {
+          context: {
+            stages: {
+              threatmetrix: {
+                transaction_id: 1,
+              },
+              resolution: {
+                transaction_id: 'resolution-mock-transaction-id-123',
+                vendor_name: 'ResolutionMock',
+              },
+              residential_address: {
```

Comment:
Is there a reason this is just called `residential_address` and not something like `resolution_residential_address`? I had to go look in progressive proofer to make sure they were parallel. Once again not necessarily for this PR, but a naming change to consider.

Reply:
the stage is named `residential_address` currently, i think it was named that way to reflect the stage that already existed named `state_id` - just the name of what was proofed.

Reply:
Seems like the whole naming scheme could use some attention at some point, although the 50/50 state makes it a hassle to change.